### DISCUSSION TITLE
一些补充问题

### DISCUSSION THREAD

**Discussion by VCStardust** [CollegesChat/university-information#75] [Enhancement]
一些对校园生活有**一定**影响的问题
_（于是这个问题集略长...）_

***

### 宿舍
1. 几人间？布局？大小？基建(水泥地/木板/瓷砖/...)？家具质量？储物空间？桌子大小？
2. 最高几层？电梯？
3. 阳台（公共大阳台？每间一个？空间大小？）
4. 停电/停水/维修频率？
5. 女生能进男生宿舍吗？男生能进女生宿舍吗？
6. 异性维修人员/保洁/宿管？
7. 宿舍信号？网速？

### 教学楼/图书馆
1. 有多少教学楼图书馆？开放时间？研讨室？
2. 图书馆好抢座吗？
3. 有看空闲教室的软件/网页吗？
4. 可以预约空教室/图书馆座位/研讨室吗？
5. 教学楼最高几层？有电梯吗？
6. 宿舍和各楼距离？
7. 体育馆/大礼堂？
8. 有专门的实验楼吗？条件如何？

### 校园生活
1. 空调/风扇覆盖范围(宿舍/教学楼/图书馆/...)？费用？
2. 学生会/社团活动多吗？有趣吗？官僚主义重吗？建议加入吗？
3. 校园活动如何？校庆？
4. 军训什么时候？水吗？
5. 校园网收费吗？怎么收费？网速/质量怎么样？覆盖率？
6. 校**内**网 必须要用 校**园**网 登录吗？有 webvpn 吗？
7. 数据/5G覆盖情况？信号质量？速度？
8. 有校内美食/商业街吗？
9. 查宿舍卫生吗？
10. 男女比例？
11. 校区位置？周围环境？

### 学业
1. 有大牛教授吗（会给本科生讲课吗）？科研成果？
2. 绩点考核/评价方式？
3. 学校风气(卷)？
4. 保研率？保研方法？转专业情况？
5. 体育要求？体测？
6. 课程安排？最晚几点下课？
7. 奖学金/助学金/勤工助学？
8. 境内境外交流机会？研究所？
9. 最好的专业？
10. 实习机会？打工？

### 周边(环境)
1. 在大学城吗？（附近高校多吗）？
2. 附近好玩吗？好吃吗？消费如何？

**Comment by zzt741** [comment]
太全了hhh，几人间确实很重要，强烈建议！宿舍大小，储物空间，桌子大小也挺重要的。

**Comment by Oneton429** [comment]
嗯 男女比例也是
挺有用的

**Reply by rvvksvnmh** [reply]
还有：学校会不会统一订购课本，学生能够自己购买二手课本
